src/widget/wknob.cpp

```diff
@@ -57,6 +57,16 @@ void WKnob::guiTick() {
 }
 
 void WKnob::inputActivity() {
+    // Bug #1793015: Previously we would simply call QWidget::update in response
```

Comment:
Just a bug number assumes we'll be using the same bug tracker forever. A URL to the bug would be better.

Reply:
Eh, URLs are pretty verbose though. When that happens, we can `s/Bug #/lp:/g` or something.

Reply:
Sure, as long as it's clearly referencing Launchpad and not a contextless number.